airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -1514,6 +1515,8 @@ def insert_job(
             characters. If not provided then uuid will be generated.
         :param project_id: Google Cloud Project where the job is running
         :param location: location the job is running
+        :param is_async: specify whether to insert job in asynchronous mode
+        :type is_async: bool
```

Comment:
```suggestion
```
The `:type:` directives are no longer needed. Sphinx can now directly infer from the type annotations rather than the docstrings for the API documentation.

Reply:
this is fixed now